app/views/managers/_form.html.erb

```diff
@@ -12,21 +12,31 @@
   <% end %>
 
   <div class="my-5">
-      <%= form.label :canvas_course_complete_date, class: "fancy_label" %>
-      <%= form.date_field :canvas_course_complete_date, class: "input_text_field" %>
-    </div>
+    <%= form.label :first_name, class: "fancy_label" %>
```

Comment:
again, is this relevant to this PR for updating the ldap_lookup gem? If not it should be in a seprate PR

Reply:
Yes, it is relevant. If the ldap_lookup gem doesn't return first and last names, they should be updated in the manager's record manually. GitHub shows the changes in this file kind of funny. All that was done: fields for first_name and last_name were added. But they were added to the top of the form and GitHub got confused a little.